app/components/options_context/options_context.ios.js

```diff
@@ -27,34 +27,34 @@ export default class OptionsContext extends PureComponent {
         };
     }
 
+    handleHide = () => {
+        this.isShowing = false;
+        this.props.toggleSelected(false, true);
+    };
+
     handleHideUnderlay = () => {
         if (!this.isShowing) {
-            this.props.toggleSelected(false, false);
+            this.props.toggleSelected(false, true);
         }
     };
 
     handleShowUnderlay = () => {
-        this.props.toggleSelected(true, false);
-    };
-
-    handleHide = () => {
-        this.isShowing = false;
-        this.props.toggleSelected(false, this.props.getPostActions().length > 0);
```

Comment:
Did we used to have a second argument on that function? That's concerning if we had one that wasn't properly removed since that should've broken something before, and it could break something else now if other code is calling it with the old argument.